lib/privs.c

```diff
@@ -36,13 +36,13 @@ DEFINE_MTYPE_STATIC(LIB, PRIVS, "Privilege information")
  */
 #ifdef HAVE_CAPABILITIES
 #ifdef HAVE_LCAPS
-static const bool privs_per_process;  /* = false */
-#elif defined(HAVE_SOLARIS_CAPABILITIES)
-static const bool privs_per_process = true;
-#endif
+static const bool privs_per_process = false;
 #else
 static const bool privs_per_process = true;
-#endif	/* HAVE_CAPABILITIES */
+#endif /* HAVE_LCAPS */
+#else	/* HAVE_CAPABILITIES */
+static const bool privs_per_process = false;
```

Comment:
The else branch of HAVE_CAPABILITIES was `... = true;` and now it's changed to `... = false;`. Is this intentional?

Reply:
no it was not.  thanks for catching this.  That stupid #if section was causing me problems.

Reply:
Thanks, but I'm still not sure that your new version is correct. You don't have an else branch for `HAVE_CAPABILITIES`, so if it's not defined, `privs_per_process` will not be declared.

I suppose it should be something like:
```
 #if defined(HAVE_CAPABILITIES) && defined(HAVE_LCAPS)
 static const bool privs_per_process;  /* = false */
 #else
 static const bool privs_per_process = true;
 #endif /* defined(HAVE_CAPABILITIES) && defined(HAVE_LCAPS) */
```